tests/api_tests.js

```diff
@@ -322,8 +323,13 @@ function runNodeEnvTests(nodeEnvData, t) {
                                          t.ok(parseInt(nodeEnvData['max.old.space.size']) > 0,
                                               "max.old.space.size is positive");
 
-                                              var nodeVersion = Number(process.version.match(/^v(\d+\.\d+)/)[1]);
-                                              if(nodeVersion >= 6.5) { // issue 283
+                                              // converts e.g. 'v8.0.0' to [8,0.0]
```

Comment:
What is the downside to using a devDependency?

If you use semver, its 2 lines of code, with no comments, and one of the lines is `var semver = require('semver')`, which barely counts as a line of code.

I can read this code, if I have to, and work through the regex, and figure if its correct, but I'd rather not.